src/nodes/display/NormalMapNode.js

```diff
@@ -57,14 +58,48 @@ class NormalMapNode extends TempNode {
 		 */
 		this.normalMapType = TangentSpaceNormalMap;
 
+		/**
+		 * Controls how to unpack the sampled normal map values.
+		 *
+		 * @type {string}
+		 * @default NoNormalPacking
+		 */
+		this.unpackNormal = NoNormalPacking;
+
 	}
 
 	setup( { material } ) {
 
-		const { normalMapType, scaleNode } = this;
+		const { normalMapType, scaleNode, unpackNormal } = this;
 
 		let normalMap = this.node.mul( 2.0 ).sub( 1.0 );
```

Comment:
This could use the `colorToDirection` function in Packing.js

That said, in the case of normal maps it may make more sense to use:

```
let normalMap = this.node.mul( 2.0 ).sub( 254.0 / 255.0 );
```

As that prevents lighting discontinuities on flat normals along uv seams where the tangents do not match exactly.